nexus/Cargo.toml

```diff
@@ -86,15 +86,19 @@ oximeter-producer.workspace = true
 [dev-dependencies]
 criterion.workspace = true
 expectorate.workspace = true
+hyper-rustls = "0.23.2"
```

Comment:
What's our policy re: putting versions in here vs. using workspace-level dep?  I would have thought the expectation was to always use a workspace dep.  Otherwise it seems like every time you add a dep here, you need to see if any other crate is using it first so you figure out if it needs to be hoisted to the workspace level?

Reply:
Using workspace deps in 705d6464b72ff25d5df490138800ac96f116fff5